tests/helper_functions.py

```diff
@@ -158,7 +158,7 @@ def pyfunc_serve_and_score_model(
     :param activity_polling_timeout_seconds: The amount of time, in seconds, to wait before
                                              declaring the scoring process to have failed.
     :param extra_args: A list of extra arguments to pass to the pyfunc scoring server command. For
-                       example, passing ``extra_args=["--no-conda"]`` will pass the ``--no-conda``
+                       example, passing ``extra_args=["--env-manager", "local"]`` will pass the ``--env-manager local``
```

Comment:
pylint complains this is too long

https://github.com/mlflow/mlflow/runs/5830151998?check_suite_focus=true#step:7:20

```
tests/helper_functions.py:161:0: C0301: Line too long (119/100) (line-too-long)
```

Can you adjust the line length by wrapping it?